### PR TITLE
Issue 5491 - UI - Add rework and finish jpegPhoto functionality

### LINKED ISSUE #5491: UI - Add rework and finish jpegPhoto functionality
**Description**
Support of binary values, specifically jpegPhoto (and displaying it in the UI) was started in the original code merge from Teko, but it is not complete and should be finished.

### PR DESCRIPTION
**Pull request by droideck**
Description: Finish the support of binary values, specifically jpegPhoto
(and displaying it in the UI) was started in the original code merge from Teko.
Disable conflicting eslint warnings.
Fix multivalued entry search in LDAP browser.

Fixes: https://github.com/389ds/389-ds-base/issues/5491

Reviewed by: ?

### CODE REVIEW

**Comment by droideck** [comment]
Replaced the plain text with a label. Please, review. :)
image

File: src/cockpit/389-console/src/lib/ldap_editor/lib/utils.jsx [hunk 1]
```diff
@@ -382,33 +388,23 @@ export function getBaseLevelEntryAttributes (serverId, baseDn, entryAttributesCa
...
       lines.map(currentLine => {
         if (currentLine !== '') {
-          entryArray.push(splitAttributeValue(currentLine));
+          if (currentLine.length < 1000 || currentLine.substring(0, 9).toLowerCase().startsWith("jpegphoto")) {
+            entryArray.push(splitAttributeValue(currentLine));
+          } else {
+            const myTruncatedValue = (<div>
+                                          currentLine.substring(0, 1000)
+                                          <Label icon={<InfoCircleIcon />} color="blue" >
+                                              Attribute is too large. Output was truncated.
```
Comment (mreynolds389) [inline src/cockpit/389-console/src/lib/ldap_editor/lib/utils.jsx]: Output is not truncated :-)  It's not displayed at all.  Maybe something like "Value is too large to display"

Comment (droideck) [reply src/cockpit/389-console/src/lib/ldap_editor/lib/utils.jsx]: Oh, right, my copy bad)
I'll fix it.

**Comment by droideck** [comment]
Done, please review:)

**Review by mreynolds389 — APPROVED** [review]
Great job!  Ack!

**Comment by droideck** [comment]
> Great job! Ack!

Thanks!

**Pull request merged**